src/main/java/com/dabsquared/gitlabjenkins/GitLabNoteRequest.java

```diff
@@ -0,0 +1,82 @@
+package com.dabsquared.gitlabjenkins;
+
+import com.dabsquared.gitlabjenkins.data.Note;
+import com.dabsquared.gitlabjenkins.data.MergeRequest;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.apache.commons.lang.builder.ToStringStyle;
+import org.gitlab.api.GitlabAPI;
+import org.gitlab.api.models.GitlabCommitStatus;
+import org.gitlab.api.models.GitlabProject;
+
+import java.io.IOException;
+
+public class GitLabNoteRequest extends GitLabRequest {
+
+	public static GitLabNoteRequest create(String payload) {
+        if (payload == null) {
+            throw new IllegalArgumentException("payload should not be null");
+        }
+
+        GitLabNoteRequest noteRequest =  Builder.INSTANCE.get().fromJson(payload, GitLabNoteRequest.class);
```

Comment:
Why storing the noteRequest object in variable if the object is immediatly returned?